src/conversion/downcast-converters.js

```diff
@@ -537,8 +534,9 @@ export function insertText() {
 			return;
 		}
 
+		const viewWriter = conversionApi.writer;
 		const viewPosition = conversionApi.mapper.toViewPosition( data.range.start );
-		const viewText = new ViewText( data.item.data );
+		const viewText = viewWriter.createText( data.item.data );
```

Comment:
`viewWriter` mixed with `conversionApi.writer`. Let's use one :P

Reply:
Okay it seems that I was who mixed things :) In this function we use only `viewWriter`. But other uses `conversionApi.writer` directly.